pkg/GAPJulia/JuliaInterface/julia/gap.jl

```diff
@@ -184,3 +184,12 @@ macro g_str(str)
 end
 
 export @g_str
+
+function GAP_help_string( topic::String, onlyexact::Bool=false )
+    return GAP.gap_to_julia( GAP.Globals.HelpString(
+           GAP.julia_to_gap( topic ), onlyexact ) )
+end
+
+function show_GAP_help( topic::String, onlyexact::Bool=false )
```

Comment:
Perhaps run VS Code autoformatter on this code section? I'd also move it into the new `help.jl`.